docs/quick-start.md

```diff
@@ -127,19 +127,19 @@ import "react-contexify/dist/ReactContexify.css";
 ### Assign an id to our menu
 
 Each `Menu` is required to have an `id`. The `id` is used to identify the `Menu` and to display it. 
-This comes in handy when your app has multiple `Menu`.
+This comes in handy when your app has multiple `Menu`s.
 
-Let's store it in a variable to avoid any typo.
+Let's store it in a variable to avoid any typos.
 ```jsx
 const MENU_ID = "menu-id";
 ```
 
 :::info
-- Only strings are supported for the moment. 
```

Comment:
Makes it sound like that's changing soon; not sure if that's what you intended. I like to avoid predicting or telegraphing the future and just update the docs if something changes. Just a personal preference!